front/app/src/Component/ConnectionState/DisplayAnonymousConnection.tsx

```diff
@@ -0,0 +1,247 @@
+/* eslint-disable curly */
+/* eslint-disable max-statements */
+/* eslint-disable max-lines-per-function */
+
+/**
```

Comment:
Did you want to comment it and then forgot ?

Reply:
I dont understand this comment ^^